src/SciChartBlazor/Modifiers/LegendModifier.cs

```diff
@@ -27,7 +27,6 @@ public class LegendModifier : ModifierBase
     /// <summary>
     /// Sets the initial orientation of the legend.
     /// </summary>
-    [PureIntEnum]
```

Comment:
Sorry I wasn't clear. Don't remove this, you need this for it to work. 

You just don't need to define the numbers in the Enum itself, and it looks cleaner. You should only define them if they aren't sequential or don't start at 0.

Reply:
Sorry, it was my mistake as well. "Tested" after removing it that it would be still horizontal and no errors, but actually vertical broke because of this.